docs/source/checks/vision/data_integrity/plot_property_label_correlation.py

```diff
@@ -38,14 +40,14 @@
 -----------------------------
 
 Image properties with a high predictive score can indicate that there is a bias in the dataset, as a single property
-can predict the label successfully using simple classic ML algorithms.
+can be used to predict the label successfully (e.g. using simple classic ML algorithms).
 
 This means that a deep learning algorithm may accidentally learn these properties instead of more accurate complex
 abstractions. For example, in the dataset of wolves and dogs photographs, the brightness of the image may be used to
 predict the label "wolf" easily.
 
 
-How do we calculate for different vision tasks?
+How do we calculate for the predictive power different vision tasks?
 -----------------------------------------------
```

Comment:
```suggestion
-------------------------------------------------------------------------
```
otherwise there will be a warning